site/ko/beta/tutorials/generative/dcgan.ipynb

```diff
@@ -0,0 +1,989 @@
+    "\n",
+    "    gradients_of_generator = gen_tape.gradient(gen_loss, generator.trainable_variables)\n",
+    "    gradients_of_discriminator = disc_tape.gradient(disc_loss, discriminator.trainable_variables)\n",
+    "\n",
+    "    generator_optimizer.apply_gradients(zip(gradients_of_generator, generator.trainable_variables))\n",
+    "    discriminator_optimizer.apply_gradients(zip(gradients_of_discriminator, discriminator.trainable_variables))"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "2M7LmLtGEMQJ"
+   },
+   "outputs": [],
+   "source": [
+    "def train(dataset, epochs):\n",
+    "  for epoch in range(epochs):\n",
+    "    start = time.time()\n",
+    "\n",
+    "    for image_batch in dataset:\n",
+    "      train_step(image_batch)\n",
+    "\n",
+    "    # GIF를 위한 이미지를 바로 생성합니다.\n",
+    "    display.clear_output(wait=True)\n",
+    "    generate_and_save_images(generator,\n",
+    "                             epoch + 1,\n",
+    "                             seed)\n",
+    "\n",
+    "    # 에포크 15개가 지날 때마다 모델을 저장합니다.\n",
+    "    if (epoch + 1) % 15 == 0:\n",
+    "      checkpoint.save(file_prefix = checkpoint_prefix)\n",
+    "    \n",
+    "    # print (' 에포크 {} 에서 걸린 시간은 {} 초 입니다'.format(epoch +1, time.time()-start))\n",
+    "    print ('Time for epoch {} is {} sec'.format(epoch + 1, time.time()-start))\n",
+    "\n",
+    "  # 마지막 에포크가 끝난 후 생성합니다.\n",
+    "  display.clear_output(wait=True)\n",
+    "  generate_and_save_images(generator,\n",
+    "                           epochs,\n",
+    "                           seed)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "2aFF7Hk3XdeW"
+   },
+   "source": [
+    "**이미지 생성 및 저장**\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "RmdVsmvhPxyy"
+   },
+   "outputs": [],
+   "source": [
+    "def generate_and_save_images(model, epoch, test_input):\n",
+    "  # `training`이 False로 맞춰진 것을 주목하세요.\n",
+    "  # 이렇게 하면 모든 층들이 추론 모드 (배치정규화)로 실행되어집니다. \n",
+    "  predictions = model(test_input, training=False)\n",
+    "\n",
+    "  fig = plt.figure(figsize=(4,4))\n",
+    "\n",
+    "  for i in range(predictions.shape[0]):\n",
+    "      plt.subplot(4, 4, i+1)\n",
+    "      plt.imshow(predictions[i, :, :, 0] * 127.5 + 127.5, cmap='gray')\n",
+    "      plt.axis('off')\n",
+    "\n",
+    "  plt.savefig('image_at_epoch_{:04d}.png'.format(epoch))\n",
+    "  plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab_type": "text",
+    "id": "dZrd4CdjR-Fp"
+   },
+   "source": [
+    "## 모델 훈련\n",
+    "위에 정의된 `train()` 메서드를 생성자와 감별자를 동시에 훈련하기 위해 부릅니다. 생성적 적대 신경망을 학습하는 것은 매우 까다로울 수 있습니다. 생성자와 감별자가 서로를 제압하지 않는 것이 중요합니다. (예시로 같은 비율로 학습하는 경우.)\n",
```

Comment:
'call' is often translated as '호출'.
I suggest '부릅니다' --> '호출합니다'

'similar rate' means speed.
I suggest '(예시로 같은 비율로 학습하는 경우.)' --> '(예를 들면 훈련 속도가 비슷해야 합니다)'

Please add one more a return char for line break.

Reply:
The original sentence in the tutorial is the following:
`It's important that the generator and discriminator do not overpower each other (e.g., that they train at a similar rate).`
As far as I know, the learning rate should be different to avoid G and D overpowering each other. What I have understood from this sentence was, if G and D are trained at a similar rate, they will overpower each other. 
So translation inside parentheses should entail such info. 
How about (예를 들면 훈련 속도가 비슷하면 서로 제압하게 됩니다.)

Reply:
Oh, I miss it. You are right. 
I just want to translate 'rate' as '학습률' and avoid repeated '제압'. :)
How about "(예를 들어 학습률이 비슷하면 한쪽이 우세해집니다.)"?

Reply:
sounds good to me!
i will change accordingly :)